### PR TITLE
Drift v2 part1 update betterproto

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

File: bxsolana/provider/ws.py [hunk 1]
```diff
@@ -71,8 +71,13 @@ async def _unary_unary(
         deadline: Optional["Deadline"] = None,
         metadata: Optional["_MetadataLike"] = None,
     ) -> "T":
+
+        request_dict = request.to_dict(include_default_values=False)
+        if "clientOrderId" in request_dict:
```
Comment (DimaCantemir) [inline bxsolana/provider/ws.py:78]: why are we removing the clientOrderId from the request ?

Comment (tinystarinagalaxy) [reply bxsolana/provider/ws.py:78]: we are not, we replace it clientOrderID

**Review by DimaCantemir — APPROVED** [review]

File: bxsolana/provider/ws.py (hunk 1 above)
Comment (kdonthi) [reply bxsolana/provider/ws.py:78]: Where?

Comment (DimaCantemir) [reply bxsolana/provider/ws.py:78]: @kdonthi clientOrderId -> clientOrderID

**Pull request merged**